### PR TITLE
Update optimize-sql-al-Database-methods-and-performance-on-server.md

### PR DESCRIPTION
**Pull request by NKarolak**
Fixed AL specific code formatting and typos after SetFilter.

Already community-checked: https://twitter.com/KarolakNatalie/status/1493536375735562241?s=20&t=BnNC8_ZZKlduYbkmOJVcUQ

### CODE REVIEW

**Review by miljance — COMMENTED** [review]
Looking good from my side. I did not even know you can format the code as AL.

**Comment by edupont04** [comment]
Thank you, @NKarolak and @miljance, for this contribution!  My team mates are on vacation (schools in most of Denmark are closed this week, the rest will be closed next week), and I don't want to accept the PR without having an "all clear" vote from either of them. Stay tuned!

**Comment by NKarolak** [comment]
> I did not even know you can format the code as AL.

@miljance The syntax highlighting for AL is even supported on GitHub :-)

File: dev-itpro/administration/optimize-sql-al-Database-methods-and-performance-on-server.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ Any result set that is returned from a call to the Find methods discussed in the
   
 The following code shows how records are most efficiently retrieved. **FindSet** is the most efficient method to use because this example reads all records.  
   
-```  
-if FindSet then  
-  repeat  
-    // Insert statements to repeat.  
-  until Next = 0;  
+```AL  
+if FindSet() then
```
Comment (PhDuck) [inline dev-itpro/administration/optimize-sql-al-Database-methods-and-performance-on-server.md:50]: Aren't we also recommending not using implicit recs?

Comment (NKarolak) [reply dev-itpro/administration/optimize-sql-al-Database-methods-and-performance-on-server.md:50]: Do you mean to use `Rec.`? This is true on most objects, except on (the same) table object.
For the sake of readability within the article, I'd prefer to omit `Rec`.

Comment (miljance) [reply dev-itpro/administration/optimize-sql-al-Database-methods-and-performance-on-server.md:50]: I think it is generally not so good practice to use the FindSet/repeat/until in the same table and I am guessing/hoping it is also not happening that much in tables. Using Rec. before FindSet sounds as a more correct pattern to me indeed.

**Review by SusanneWindfeldPedersen — APPROVED** [review]

**Comment by SusanneWindfeldPedersen** [comment]
Taking in this change. Thank you so much for the contribution - your work is much appreciated and helps others as well as us internally, so thank you!

**Pull request merged**